src/zenml/orchestrators/local_docker/local_docker_orchestrator.py

```diff
@@ -153,8 +153,39 @@ def submit_pipeline(
         environment[ENV_ZENML_LOCAL_STORES_PATH] = local_stores_path
         start_time = time.time()
 
+        # Get execution mode for failure handling
```

Comment:
```suggestion
```

I'll suggest removing all of those comments if they provide no value and just make it more effort to read the code. In general, if a variable is called `execution_mode`, I think its pretty clear that this is fetching the execution mode 😄